src/renderer/components/ft-checkbox-list/ft-checkbox-list.vue

```diff
@@ -0,0 +1,33 @@
+<template>
+  <div class="pure-radiobutton filter">
```

Comment:
Looks like the CSS file contains stuff for both radio buttons and checkboxes but you are using some radio button classes here, is that intentional? If yes, could you please remove the unused CSS from the CSS file.

Reply:
Oh yeah, I forgot to leave a comment about the css. This is the same css we use for the radio buttons css file. I wasn't sure if it'd be better to use the same css file for both or remove checkbox stuff from the radio button file and radio button stuff from the checkbox file. Do you think it'd be better to use the same file for both?

Reply:
I think it might be best to keep the CSS separated to avoid future confusion about why there is a lot of CSS in the file of a component that doesn't use a lot of it.

That being said the `ft-checkbox-list` Vue component seems to be using just `radiobutton` classes which is a bit confusing.